src/axom/quest/Shaper.cpp

```diff
@@ -313,5 +312,25 @@ double Shaper::allReduceSum(double val) const
   return val;
 }
 
+double Shaper::allReduceMin(double val) const
```

Comment:
Future candidates for ``axom::core``? Any chance the 2 returns result in a warning about unreachable code?

Reply:
Oh -- good call -- that should have been in a `#else` -- I'll fix them.